css/trac.css

```diff
@@ -1,21 +1,37 @@
 #trac {
 	padding: 140px 30px 20px 30px;
 	margin: 0 0 0 0;
+	color: #444;
 }
-#trac table td{
+
+#trac table table {
+	margin: 5px;
+	border-collapse: collapse;
+	color: #DDD;
+}
+
+#trac table table, table table td{

```

Comment:
Shouldn't there be a `#trac` after the comma? Otherwise, this will be applied to all `table table td`s. Is this intended?

Reply:
ah yes, my bad there should be `#trac`